ui/src/storage/StorageItemView.vue

```diff
@@ -2,10 +2,19 @@
   <div>
     <div v-if="item !== null">
```

Comment:
I think this should be "item !== undefined" 

Maybe that does the trick? For me the forced update looks rather strange - it should actually work without, this is not a too special case I think.

Reply:
That should be done as well, but it only changed, such that the LinearProgressIndicator is displayed the whole time
![image](https://user-images.githubusercontent.com/52160939/141254754-e7533ce8-e134-45d6-af3e-06fe5c52b2e9.png)